contracts/lib/TokenTransferrerConstants.sol

```diff
@@ -33,6 +33,7 @@ pragma solidity >=0.8.7;
  *      codebase but have been left in for readability.
  */
 
+uint256 constant AlmostOneWord = 0x19;
```

Comment:
Huh shouldn't this be 31? 0x1f instead

Reply:
how about `OneWord -1` ?

Reply:
better!

Reply:
Actually doesn't compile:
```solidity
TypeError: Only direct number constants and references to such constants are supported by inline assembly.
   --> contracts/lib/TokenTransferrer.sol:662:51:
    |
662 |                             add(returndatasize(), AlmostOneWord),
    |                                                   ^^^^^^^^^^^^^
```
So I kept it as 0x1f in the PR
https://github.com/ProjectOpenSea/seaport/pull/388